manifests/kots-config.yaml

```diff
@@ -95,8 +92,8 @@ spec:
     - name: metadataDatabase
       title: Metadata Database
       description: |-
-        Configure the information needed to connect to your metadata database.
-        If you need further info, please check [docs.carto.com](https://docs.carto.com/~/changes/1Lia4Pv0PjH33F9R0CyP/carto-self-hosted/deployment-guides/kubernetes#metadata-database).
+        Information needed to connect to your metadata database.
+        [More info](https://docs.carto.com/carto-self-hosted/key-concepts/deployment-requirements#external-database).
```

Comment:
ey, why you delete the custom branch from here? :thinking:

Reply:
Just a mistake